src/main/java/com/indeed/jiraactions/api/IssuesAPICaller.java

```diff
@@ -132,17 +134,16 @@ private String getBasicInfoURL() throws UnsupportedEncodingException {
         return url;
     }
 
-    protected static final DateTimeZone JIRA_TIME_ZONE = DateTimeZone.forID("America/Chicago");
     private static final DateTimeFormatter JIRA_TIME_FORMAT = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm");
     /**
      * Imhotep builders always runs in UTC-6, regardless of DST. However, Jira observes daylight savings time.
      * So when we're in DST and run for "2018-04-01" to "2018-04-02", we need to tell Jira we actually mean
```

Comment:
Can you add a unit test case for this? (maybe add back in the deleted TestsIssuesApiCaller?)